### DISCUSSION TITLE
Customize Profile Activity Color

### DISCUSSION THREAD

**Discussion by jlosito** [community/community#42313] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In the recent months, GitHub has changed users' activity from the traditional green to orange during October and blue during December. I personally get really excited when these changes get rolled out. That got me thinking it would be awesome if users could customize their profile's activity color to whatever color they want throughout the year and not only during special occasions. Similarly how the user's README gives their profile a personal touch, it would be cool if their activity color also provided their personal touch.

**Comment by Luis-Angel-Santos** [comment]
I agree with you, as you say it would be an excellent way to give your personal style in your GitHub perifl. 👍🏻

Hopefully in the not too distant future they can implement that and more things to customize our profiles.
